.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,45 @@
+name: 🐛 Bug report
+description: Something is not working correctly.
+title: "🐛 <title>"
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        **Thank you for wanting to report a bug for this project!**
+
+        ⚠
+        Verify first that your issue is not [already reported on GitHub](https://github.com/cda-tum/mqt-core/search?q=is%3Aissue&type=issues).
+
+        If you have general questions, please consider [starting a discussion](https://github.com/cda-tum/mqt-core/discussions).
+  - type: textarea
+    attributes:
+      label: Environment information
+      description: >-
+        Please provide information about your environment. For example, OS, C++ compiler, mqt.core version etc.
+      placeholder: |
+        - OS:
+        - C++ compiler:
+        - mqt.core version:
+        - Additional environment information:
```

Please never blindly copy and paste code. This contains quite some references to mqt-core
```suggestion
        Verify first that your issue is not [already reported on GitHub](https://github.com/cda-tum/mqt-qudits/search?q=is%3Aissue&type=issues).

        If you have general questions, please consider [starting a discussion](https://github.com/cda-tum/mqt-qudits/discussions).
  - type: textarea
    attributes:
      label: Environment information
      description: >-
        Please provide information about your environment. For example, OS, C++ compiler, mqt.core version etc.
      placeholder: |
        - OS:
        - C++ compiler:
        - mqt.qudits version:
        - Additional environment information:
```